core/src/main/java/com/graphhopper/routing/weighting/TurnWeighting.java

```diff
@@ -108,22 +111,14 @@ public double calcTurnWeight(int edgeFrom, int nodeVia, int edgeTo) {
         if (!EdgeIterator.Edge.isValid(edgeFrom) || !EdgeIterator.Edge.isValid(edgeTo)) {
             return 0;
         }
+        double tCost = 0;
         if (turnCostExt.isUTurn(edgeFrom, edgeTo)) {
-            // note that the u-turn costs overwrite any turn costs set in TurnCostStorage
-            // todo:
-            // also this does not allow TurnCostEncoder to set the u-turn costs to zero explicitly, like FootFlagEncoder!
-            // this problem is a bit hidden, because if you do not apply any turn restrictions (like FootFlagEncoder)
-            // you never do a u-turn anyway.
```

Comment:
the comment still applies, no ? setting u-turn costs on turnCostExt has no effect, because the uTurnCosts take a higher priority here.

Reply:
Ah, ok. This is what was meant with the comment before the todo. I'll revert the removal of that.

Regarding the TODO: the FootFlagEncoder does not have any turn restriction stuff but still overwriting/overruling is not possible. 

BTW: could we make isUTurn dependent on turnCostEnc and then remove this comment?